elastic/beats
# Use server-side filtering when retrieving Cloud Foundry logs

## What does this PR do?

Use server-side filtering when retrieving Cloud Foundry logs.

## Why is it important?

It reduces CPU usage of Filebeat significatively.

## Checklist




## Author's Checklist



## How to test this PR locally



* Run Cloud Foundry input with `firehose_filter: logs`, check that log messages are still collected.

## Related issues


- 

## Use cases



## Screenshots



## Logs
